src/Refinery/To/Transformation/DateTimeTransformation.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/* Copyright (c) 2018 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Refinery\To\Transformation;
+
+use ILIAS\Data\Factory;
+use ILIAS\Data\Result;
+use ILIAS\Refinery\Transformation;
+
+/**
+ * Transform value to php \DateTimeImmutable
```

Comment:
Which kind of value? Looking at your tests this would be a common known date or date time string. Like `04.07.2019 15:30:24` right? Maybe we should this behavior as description here.

Reply:
Maybe a short link to https://www.php.net/manual/de/datetime.formats.php is sufficient here?